packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss

```diff
@@ -331,11 +341,22 @@
   }
 
   .ag-cell {
+    display: inline-flex;
```

Comment:
I suspect this would cause issue. Grid cells are designed to be not flex based, e.g. text alignment could have issue. And this would be a soft breaking change for users migrate from existing ag grid theme.

Reply:
will investigate an alternative